sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_operations/_operations.py

```diff
@@ -0,0 +1,746 @@
+    ) -> bytes:
+        """Get a registered schema by its unique ID reference.
+
+        Gets a registered schema by its unique ID.  Azure Schema Registry guarantees that ID is unique
+        within a namespace. Operation response type is based on serialization of schema requested.
+
+        :param id: Schema ID that uniquely identifies a schema in the registry namespace. Required.
+        :type id: str
+        :keyword bool stream: Whether to stream the response of this operation. Defaults to False. You
+         will have to context manage the returned stream.
+        :return: bytes
+        :rtype: bytes
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        error_map = {
+            401: ClientAuthenticationError, 404: ResourceNotFoundError, 409: ResourceExistsError, 304: ResourceNotModifiedError
+        }
+        error_map.update(kwargs.pop('error_map', {}) or {})
+
+        _headers = kwargs.pop("headers", {}) or {}
+        _params = kwargs.pop("params", {}) or {}
+
+        cls: ClsType[bytes] = kwargs.pop(
+            'cls', None
+        )
+
+        
+        request = build_schema_registry_get_schema_by_id_request(
+            id=id,
+            api_version=self._config.api_version,
+            headers=_headers,
+            params=_params,
+        )
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, 'str', skip_quote=True),
+        }
+        request.url = self._client.format_url(request.url, **path_format_arguments)
+
+        _stream = kwargs.pop("stream", False)
+        pipeline_response: PipelineResponse = self._client._pipeline.run(   # pylint: disable=protected-access
+            request,
+            stream=_stream,
+            **kwargs
+        )
+
+        response = pipeline_response.http_response
+
+        if response.status_code not in [200]:
+            if _stream:
+                 response.read()  # Load the body in memory and close the socket
+            map_error(status_code=response.status_code, response=response, error_map=error_map)
+            raise HttpResponseError(response=response)
+
+        response_headers = {}
+        response_headers['location']=self._deserialize('str', response.headers.get('location'))
+        response_headers['schema-id']=self._deserialize('str', response.headers.get('schema-id'))
+        response_headers['schema-id-location']=self._deserialize('str', response.headers.get('schema-id-location'))
+        response_headers['schema-group-name']=self._deserialize('str', response.headers.get('schema-group-name'))
+        response_headers['schema-name']=self._deserialize('str', response.headers.get('schema-name'))
+        response_headers['schema-version']=self._deserialize('int', response.headers.get('schema-version'))
+
+        if _stream:
+            deserialized = response.iter_bytes()
+        else:
+            deserialized = _deserialize(
```

Comment:
this raises the following error:
```  File "C:\Users\swathip\repos\azure-sdk-for-python\sdk\schemaregistry\azure-schemaregistry\azure\schemaregistry\_model_base.py", line 684, in _deserialize_with_callable
    return typing.cast(typing.Callable[[typing.Any], typing.Any], deserializer)(value)    
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^    
  File "C:\Users\swathip\repos\azure-sdk-for-python\sdk\schemaregistry\azure-schemaregistry\azure\schemaregistry\_model_base.py", line 255, in _deserialize_bytes
    return bytes(base64.b64decode(attr))
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\swathip\AppData\Local\Programs\Python\Python311\Lib\base64.py", line 88, in b64decode
    return binascii.a2b_base64(s, strict_mode=validate)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
binascii.Error: Incorrect padding```

Wonder if this has to do with the utf8_decode bug?

Reply:
needed to pass stream=True into client call